lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -34,6 +35,47 @@ import 'window.dart';
 ui.VoidCallback? scheduleFrameCallback;
 
 typedef _KeyDataResponseCallback = void Function(bool handled);
+typedef HighContrastListener = void Function(bool);
```

Comment:
Consider giving the `bool` parameter a name (see `handled` in `_KeyDataResponseCallback` above).

Consider documenting public members of the library (the typedef `HighContrastListener` is public).